doc/index.md

```diff
@@ -230,9 +228,9 @@ defaults?: {
     };
     videoDeviceId?: string | null;
     audioDeviceId?: string | null;
-    micAutoGainControl?: ConstrainBoolean | boolean;
-    micEchoCancellation?: ConstrainBoolean | boolean;
-    micNoiseSuppression?: ConstrainBoolean | boolean;
+    micAutoGainControl?: ConstrainBoolean;
+    micEchoCancellation?: ConstrainBoolean;
+    micNoiseSuppression?: ConstrainBoolean;
```

Comment:
This was my fault in your last PR. I didn't realize that `ConstrainBoolean` was already 
```ts
type ConstrainBoolean = boolean | ConstrainBooleanParameters;
```